opentelemetry-api/src/opentelemetry/trace/propagation/__init__.py

```diff
@@ -13,22 +13,24 @@
 # limitations under the License.
 from typing import Optional
 
-from opentelemetry import trace as trace_api
 from opentelemetry.context import get_value, set_value
 from opentelemetry.context.context import Context
+from opentelemetry.trace.span import INVALID_SPAN, Span
 
 SPAN_KEY = "current-span"
 
 
 def set_span_in_context(
-    span: trace_api.Span, context: Optional[Context] = None
```

Comment:
Can we have some doc-strings for these methods? Just a summary of what will be done exactly.

Reply:
added.